data/json/items/melee/bludgeons.json

```diff
@@ -336,13 +336,32 @@
     "symbol": "/",
     "material": [ "wood", "aluminum" ],
     "techniques": [ "WBLOCK_1" ],
-    "flags": [ "NONCONDUCTIVE" ],
+    "flags": [ "NONCONDUCTIVE", "CRUTCHES" ],
     "volume": "1 L",
     "longest_side": "80 cm",
     "bashing": 10,
     "price": 16000,
     "price_postapoc": 100
   },
+  {
+    "id": "crutches",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "dark_gray",
+    "name": { "str": "crutches", "str_pl": "pairs of crutches" },
+    "description": "A pair of crutches",
+    "price": 2500,
+    "material": [ "aluminum" ],
+    "techniques": [ "WBLOCK_1", "SWEEP" ],
+    "flags": [ "CRUTCHES" ],
+    "weight": "2200 g",
+    "volume": "1500 ml",
+    "longest_side": "150 cm",
+    "bashing": 24,
```

Comment:
This feels too overpowered. Baseball bats has bashing 22, police baton has bashing 21. I don't think aluminum crutches should deliver that much bashing damage.